### PR TITLE
feat(coinmarket): RBF in exchange

### LINKED ISSUE #9938: Enable RBF for exchange transactions in coinmarket
**Describe the bug**
Users are unable to bump fees for Exchange transaction made with BTC.

### PR DESCRIPTION
**Pull request by FreeWall**
## Description

Let’s send all Exchange transactions with RBF on. User will be able to bump the tx fee in Suite account.
(For sell we’re unable to allow this since the Sell providers require txid which changes after the fees are updated)

## Related Issue

Resolve #9938

## Screenshots:
image

### CODE REVIEW

File: packages/suite/src/hooks/wallet/useCoinmarketRecomposeAndSign.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const useCoinmarketRecomposeAndSign = () => {
             ethereumDataHex?: string,
             recalcCustomLimit?: boolean,
             ethereumAdjustGasLimit?: string,
+            options: FormOptions[] = ['broadcast'],
```
Comment (matejkriz) [inline packages/suite/src/hooks/wallet/useCoinmarketRecomposeAndSign.ts:27]: nitpick: if you put it on let's say fifth place, you can save the ugly `undefined` params

File: packages/suite/src/hooks/suite/__tests__/useCountdownTimer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}: {
+    params: Parameters<typeof useCountdownTimer>;
+    callback: (res: Result) => void;
+}) => {
+    const result = useCountdownTimer(...params);
+    callback(result);
+    return null;
+};
+
+describe('useCountdownTimer', () => {
```
Comment (matejkriz) [inline packages/suite/src/hooks/suite/__tests__/useCountdownTimer.test.tsx:19]: Thanks for the test. Did you add it for coverage? Please put it in separate commit, it's not related at all to RBF in exchange 🙏

File: packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts [hunk 3]
```diff
@@ -309,6 +309,10 @@ export const useOffers = ({ selectedAccount }: UseCoinmarketExchangeFormProps) =
                 selectedQuote.sendAddress,
                 sendStringAmount,
                 selectedQuote.partnerPaymentExtraId,
+                undefined,
+                undefined,
+                undefined,
+                ['broadcast', 'bitcoinRBF'],
```
Comment (matejkriz) [inline packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts:315]: Have you considered other coins than bitcoin here?

File: packages/suite/src/hooks/suite/__tests__/useCountdownTimer.test.tsx (hunk 2 above)
Comment (FreeWall) [reply packages/suite/src/hooks/suite/__tests__/useCountdownTimer.test.tsx:19]: Yes, fixed in https://github.com/trezor/trezor-suite/pull/9939/commits/3335536c4c736c4d909d0772f96d7a56c44399a7

File: packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts (hunk 3 above)
Comment (FreeWall) [reply packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts:315]: Should be fine, transaction functions check network type for bitcoinRBF.

File: packages/suite/src/hooks/wallet/useCoinmarketRecomposeAndSign.ts (hunk 1 above)
Comment (FreeWall) [reply packages/suite/src/hooks/wallet/useCoinmarketRecomposeAndSign.ts:27]: I didn't want to interfere to other calls of this function, so I added it as last optional param.

**Review by matejkriz — APPROVED** [review]
utACK
